<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhPreferNumbers</set>
  </property>
  <widget class="QFrame" name="center_frame">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>80</y>
     <width>590</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="search_order_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:20px;
font-size:15px;
</string>
    </property>
    <property name="text">
     <string>Search Order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_order_line_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:25px;
background-color: rgba(13, 153, 255,0.4);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter Order ID</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rooms_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:20px;
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Rooms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_order_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:20px;
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>New Order</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px;
</string>
    </property>
    <property name="text">
     <string>Welcome to room manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
